community/erlang/APKBUILD

```diff
@@ -1,105 +1,99 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Contributor: Gabriele Santomaggio <g.santomaggio@gmail.com>
 # Contributor: Marlus Saraiva <marlus.saraiva@gmail.com>
-# Maintainer: 
+# Maintainer: Daniel Isaksen <d@duniel.no>
 pkgname=erlang
-pkgver=21.2.6
+pkgver=22.0.2
 _srcver=$pkgver
-pkgrel=2
+pkgrel=0
 pkgdesc="General-purpose programming language and runtime environment"
 url="http://www.erlang.org/"
 license="Apache-2.0"
 arch="all"
 depends="$pkgname-kernel $pkgname-stdlib $pkgname-compiler"
-makedepends="perl-dev zlib-dev ncurses-dev openssl-dev openjdk8 unixodbc-dev
-	autoconf wxgtk-dev glu-dev"
+makedepends="perl-dev perl zlib-dev ncurses-dev openssl-dev openjdk8
+             unixodbc-dev autoconf wxgtk-dev glu-dev"
 subpackages="$pkgname-dev
              $pkgname-asn1:asn
-	     $pkgname-common-test:common_test
-	     $pkgname-compiler:compiler
-	     $pkgname-crypto:crypto
-	     $pkgname-debugger:debugger
-	     $pkgname-dialyzer:dialyzer
-	     $pkgname-diameter:diameter
-	     $pkgname-edoc:edoc
-	     $pkgname-eldap:eldap
-	     $pkgname-erl-docgen:erl_docgen
-	     $pkgname-erl-interface:erl_interface
-	     $pkgname-erts:erts
-	     $pkgname-et:et
-	     $pkgname-eunit:eunit
-	     $pkgname-hipe:hipe
-	     $pkgname-inets:inets
-	     $pkgname-jinterface:jinterface
-	     $pkgname-kernel:kernel
-	     $pkgname-megaco:megaco
-	     $pkgname-mnesia:mnesia
-	     $pkgname-observer:observer
-	     $pkgname-odbc:odbc
-	     $pkgname-os-mon:os_mon
-	     $pkgname-otp-mibs:otp_mibs
-	     $pkgname-parsetools:parsetools
-	     $pkgname-public-key:public_key
-	     $pkgname-reltool:reltool
-	     $pkgname-runtime-tools:runtime_tools
-	     $pkgname-sasl:sasl
-	     $pkgname-snmp:snmp
-	     $pkgname-ssh:ssh
-	     $pkgname-ssl:ssl
-	     $pkgname-stdlib:stdlib
-	     $pkgname-syntax-tools:syntax_tools
-	     $pkgname-tools:tools
-	     $pkgname-xmerl:xmerl
-	     $pkgname-wx:wx"
-options="!check"
+             $pkgname-common-test:common_test
```

Comment:
aports uses tabs for indentation, not spaces.

Reply:
Ah crap, newly configured editor. I'll fix that